src/main/kotlin/no/nav/syfo/behandler/BehandlerService.kt

```diff
@@ -68,47 +69,47 @@ class BehandlerService(
         }
 
         val pBehandlereForArbeidstakerList =
-            database.getBehandlerForArbeidstakerMedType(
-                personIdentNumber = behandlerArbeidstakerRelasjon.arbeidstakerPersonident,
+            database.getBehandlerAndRelasjonstypeList(
+                arbeidstakerIdent = behandlerArbeidstakerRelasjon.arbeidstakerPersonident,
             )
 
-        val isBytteAvFastlege = behandlerArbeidstakerRelasjon.type == BehandlerArbeidstakerRelasjonType.FASTLEGE &&
+        val isBytteAvFastlege = behandlerArbeidstakerRelasjon.type == BehandlerArbeidstakerRelasjonstype.FASTLEGE &&
             pBehandlereForArbeidstakerList
-            .filter { (_, behandlerType) -> behandlerType == BehandlerArbeidstakerRelasjonType.FASTLEGE.name }
+            .filter { (_, behandlerType) -> behandlerType == BehandlerArbeidstakerRelasjonstype.FASTLEGE }
             .map { (pBehandler, _) -> pBehandler.id }.firstOrNull() != pBehandler.id
 
         val behandlerIkkeKnyttetTilArbeidstaker = !pBehandlereForArbeidstakerList
-            .filter { (_, behandlerType) -> behandlerType == behandlerArbeidstakerRelasjon.type.name }
+            .filter { (_, behandlerType) -> behandlerType == behandlerArbeidstakerRelasjon.type }
             .map { (pBehandler, _) -> pBehandler.id }.contains(pBehandler.id)
 
         if (isBytteAvFastlege || behandlerIkkeKnyttetTilArbeidstaker) {
             addBehandlerToArbeidstaker(
                 behandlerArbeidstakerRelasjon = behandlerArbeidstakerRelasjon,
                 behandlerId = pBehandler.id,
             )
-        } else if (behandlerArbeidstakerRelasjon.type == BehandlerArbeidstakerRelasjonType.SYKMELDER) {
+        } else if (behandlerArbeidstakerRelasjon.type == BehandlerArbeidstakerRelasjonstype.SYKMELDER) {
             database.updateBehandlerArbeidstakerRelasjon(
                 behandlerArbeidstakerRelasjon = behandlerArbeidstakerRelasjon,
                 behandlerId = pBehandler.id,
             )
         }
 
         return pBehandler.toBehandler(
-            kontor = database.getBehandlerKontorForId(pBehandler.kontorId)
+            kontor = database.getBehandlerKontorById(pBehandler.kontorId)
         )
     }
 
     private fun getBehandler(behandler: Behandler): PBehandler? {
         return when {
-            behandler.personident != null -> database.getBehandlerMedPersonIdentForPartnerId(
-                behandlerPersonIdent = behandler.personident,
+            behandler.personident != null -> database.getBehandlerByBehandlerFnrAndPartnerId(
+                behandlerFnr = behandler.personident,
                 partnerId = behandler.kontor.partnerId,
             )
-            behandler.hprId != null -> database.getBehandlerMedHprIdForPartnerId(
-                hprId = behandler.hprId,
+            behandler.hprId != null -> database.getBehandlerByHprIdAndPartnerId(
+                HPRId = behandler.hprId,
```

Comment:
Konvensjon er vel alltid lowercase i starten?

Reply:
Denne var litt kinking, så dette ble resultatet etter en diskusjon. Det er et forsøk på å skapen et synlig skille mellom herId og hprId, det er bare én bokstav som skiller, så det er fort gjort å se feil. HPR er en forkortelse, Her-id vet vi ikke hva står for, og skrivemåten er også forskjellig av alle (herId, HerId, HerID, HER-id, osv). Samtidig er begge ganske innarbeidet, og uttales veldig forskjellig, så helsepersonellRegister føles litt fremmed. Konklusjonen vår var at lower case konvensjonen kunne sees bort fra for en god sak i dette tilfellet

Men da bør vi kanskje være mer konskvent på bruken av HPR i uppercase overalt? Dvs:

```
behandler.HPRId != null -> database.getBehandlerByHPRIdAndPartnerId(
HPRId = behandler.HPRId
```

Reply:
Kunne man brukt value object for HprId (sånn dere har gjort for PartnerId) ? For å bedre skille den fra herId uten å bryte lower case-konvensjon.

Reply:
H i `Her` står i hvert fall for `Helse`, E står `Enhet` og da antar jeg at R står for `Register`

Da liker jeg bedre forslaget til @andersrognstad.

Slik det er nå kunne det jo like godt vært `HERId` og `hprId`, og virker da tilfeldig. Hvis det f.eks hadde vært en tredje forkortelse som startet på `H` hadde man jo vært like langt. Å gå bort i fra konvesjon her mener jeg ikke er nødvendig, for dette "problemet" er allerede dekket, f.eks med dataklasse soml @andersrognstad foreslår eller med å skrive hele ordet ut som jeg foreslo over.

Reply:
Diskutert litt mer, her er vil mye løse seg med et nytt value object, men vi tar det ikke nå. Bruker hprId og herId.